centos_7_py2py3/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM amd64/centos:7
+
+MAINTAINER Cosmo (hello@cloudify.co)
+
+RUN yum -y install python-devel gcc openssl git libxslt-devel libxml2-devel openldap-devel libffi-devel openssl-devel libvirt-devel python-netaddr
+
+RUN curl --silent --show-error --retry 5 https://bootstrap.pypa.io/get-pip.py | python
+
+RUN pip install virtualenv==15.1.0
+RUN cd / && \
+    virtualenv py2env && \
+    source py2env/bin/activate && \
+    pip install --upgrade pip==9.0.1 && \
+    pip install --upgrade virtualenv==15.1.0 && \
+    pip install git+https://github.com/cloudify-cosmo/wagon.git@add-venvs-to-pip-wheel
+
+
+RUN yum -y install --enablerepo=updates python3 python3-devel
+
+RUN curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py"
+
+RUN python3 get-pip.py
+
```

Comment:
Can we do yum install python3-virtualenv for this by any chance?

Reply:
There is no such package.

Reply:
No package needed, sorry- wasn't sure if Centos/RHEL packaged it together:
python3 -m venv <path to new venv>